root/defaults/rutorrent-conf/plugins.ini

```diff
@@ -0,0 +1,36 @@
+;; Plugins' permissions.
+;; If flag is not found in plugin section, corresponding flag from "default" section is used.
+;; If flag is not found in "default" section, it is assumed to be "yes".
+;;
+;; For setting individual plugin permissions you must write something like that:
+;;
+;; [ratio]
+;; enabled = yes                        ;; also may be "user-defined", in this case user can control plugin's state from UI
+;; canChangeToolbar = yes
+;; canChangeMenu = yes
+;; canChangeOptions = no
+;; canChangeTabs = yes
+;; canChangeColumns = yes
+;; canChangeStatusBar = yes
+;; canChangeCategory = yes
+;; canBeShutdowned = yes
+
+[default]
```

Comment:
Is this needed? I need to investigate the consequences of doing this. Maybe if ipad plugin is deprecated it is a better solution to delete it in Dockerfile.

Reply:
According to the ruutorrentmobile readme the ipad plugin needs to be disabled to enable scrolling. Also the ipad plugin readme mentions that rutorrentMobile is a better solution then the ipad plugin. The ipad plugin, however, is included in the rutorrent plugin repo. So you download it as part of a larger package. We could of course remove the specific folder in the docker file. Let me know what you think is better

Reply:
Thank you, as you said I found here https://github.com/Novik/ruTorrent/wiki/PluginIPad even Novik recommends rutorrentmobile plugin, so I think it is a better idea to delete ipad plugin from the image and correct rutorrentmobile installation, and don't use plugins.ini. I am going to prepare a new release with this changes.